modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+
+.Enabling assisted installer service on bare metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
+
+For distributed units (DUs), {rh-rhacm} supports single node OpenShift deployments, which run on a single bare metal host. The single node cluster acts as both a control plane and a worker node. {rh-rhacm} supports single node OpenShift deployments for {product-title} on top of a single BMO DU.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+* Install {product-title} {product-verstion} on a hub cluster.
+* Install {rh-rhacm} and create the MultiClusterHub resource.
+* Create PersistentVolume custom resources (CR) for database and file system storage.
+* Install openshift-install CLI for the desired version of {product-title}.
+
+.Next Steps
+
+. Modify `HiveConfig` to enable the feature gate for Assisted Installer:
++
+[source,terminal]
+----
+ oc patch hiveconfig hive --type merge -p '{"spec":{"targetNamespace":"hive","logLevel":"debug","featureGates":{"custom":{"enabled":["AlphaAgentInstallStrategy"]},"featureSet":"Custom"}}}'
+----
+
+. Modify the `provisioning` resource to allow the bare metal operator to watch all namespaces:
++
+[source,terminal]
+----
+ oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true }}'
+----
+
+. Create an `AgentServiceConfig` custom resource:
++
+[source,yaml]
+----
+ kind: AgentServiceConfig
+ metadata:
+   name: agent
+   namespace: assisted-installer
+ spec:
+   databaseStorage:
+     volumeName: <db_pv_name> <1>
+     accessModes:
+     - ReadWriteOnce
+     resources:
+       requests:
+         storage: <db_storage_size> <2>
+   filesystemStorage:
+     volumeName: <fs_pv-name> <3>
+     accessModes:
+     - ReadWriteOnce
+     resources:
+       requests:
+         storage: <fs_storage_size> <4>
+   osImages:
+     - openshiftVersion: <ocp_version> <5>
+       rootfs: <rootfs_url> <6>
+       url: <iso_url> <7>
+----
+<1> PersistentVolume name to use for database storage.
```

Comment:
```suggestion
<1> The `PersistentVolume` name to use for database storage.
```

This sounds like a reference to a specific PV, so it should be styled. Check out the [object reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references) guidelines and update the rest of the annotations.

Reply:
Updated as suggested.